src/tests/test_release_factory.py

```diff
@@ -41,21 +43,20 @@ def test_init_release_default(self, test_dbs) -> None:
         with metadata_db.session_scope() as session:
             last_release = session.query(EnsemblRelease).order_by(EnsemblRelease.version.desc()).first()
             expected_version = Decimal("1.0") if last_release is None else last_release.version + Decimal("0.1")
+        label = "2028-09-11"
+        date = datetime.strptime(label, "%Y-%m-%d").date()
 
-            try:
-                # Call init_release but don't assert on the returned object
-                factory.init_release(label=str(expected_version))
-            except Exception as e:
-                pytest.fail(f"Unexpected exception: {e}")
+        try:
+            factory.init_release(label=label)
+        except Exception as e:
+            pytest.fail(f"Unexpected exception: {e}")
```

Comment:
It's been a while since I worked with unit tests, but it feels like this `try-except` is not required as if an exception is raised, pytest will stop the test, raising the exception.